var/spack/repos/builtin/packages/msmpi/package.py

```diff
@@ -69,10 +69,10 @@ def build_command_line(self):
             )
         args.append("/p:IFORT_BIN=%s" % os.path.dirname(ifort_bin))
         args.append("/p:VCToolsVersion=%s" % self.pkg.compiler.msvc_version)
-        args.append("/p:WindowsTargetPlatformVersion=%s" % str(self.pkg.spec["wdk"].version))
-        args.append("/p:PlatformToolset=%s" % self.pkg.compiler.cc_version)
+        args.append("/p:WindowsTargetPlatformVersion=%s" % str(self.pkg.spec["win-wdk"].version))
```

Comment:
This query should be expressed through the compiler, not through the WDK. In general all queries about the current platform w.r.t. everything but the version of Windows (i.e. VS version, platform toolset, targeted platform, etc) should be made through the compiler as that is what is responsible for configuring all of this in the build environment.